torch/testing/_internal/common_utils.py

```diff
@@ -1029,14 +1031,10 @@ def remove_prefix(text, prefix):
             if text.startswith(prefix):
                 return text[len(prefix):]
             return text
-        # NB: we take __file__ from the module that defined the test
-        # class, so we place the expect directory where the test script
-        # lives, NOT where test/common_utils.py lives.  This doesn't matter in
-        # PyTorch where all test scripts are in the same directory as
-        # test/common_utils.py, but it matters in onnx-pytorch
+        # Always look for 'expect' in the same directory as 'common_utils.py'.
         module_id = self.__class__.__module__
         munged_id = remove_prefix(self.id(), module_id + ".")
-        test_file = os.path.realpath(sys.modules[module_id].__file__)
+        test_file = os.path.realpath(__file__)
```

Comment:
This change doesn't make sense to me. In particular, I don't see any justification why you haven't just broken `onnx-pytorch` by changing it this way. You should revert this change and just move the expect files to the expected directories (now that you have reorganized the folder structure.)

Reply:
The `expect` directory moved to `torch/testing/_internal/expect` and the old code here no longer worked correctly since it looked for the files in the wrong directory. This codepath is used by pytorch tests and also onnx tests. Although onnx tests had a separate expect directory: `test/onnx/expect`. Instead of having separate expect folders, I merged the into one (`torch/testing/_internal/expect`) and this common component always looks in that directory for the expect files.

I feel this more consistent than having separate `expect` directories and handling the `onnx` codepath separately here.

Reply:
> Instead of having separate expect folders, I merged the into one (torch/testing/_internal/expect) and this common component always looks in that directory for the expect files.

Here's is my reasoning why there should be separate expect folders. Let me suppose that I'm adding an expect test. Typically, I'll be editing a file in `test/`, like `test/test_jit.py`. A principle that I generally like to see is that related files should be close by; so if I need to update expect tests, they should be similarly close by, like `test/expect/...`. Suppose that I had an out-of-tree test suite, like `pytorch-onnx/test`; if the test directory is based on the folder location, I can put the expect tests out-of-tree too, and not have to put them in `torch/testing/_internal/expect`. The consistency is that expect tests live with the test script which generates them!

Reply:
I see your point. I moved all the expect files back to the original location and nuked `torch.testing._internal.expect`.